test/integration/index.ts

```diff
@@ -3,12 +3,20 @@ import path from "path";
 import Mocha from "mocha";
 // eslint-disable-next-line import/no-extraneous-dependencies
 import glob from "glob";
+import { sleep } from "./helpers/sleep";
 
 export function run(): Promise<void> {
   // Create the mocha test
   const mocha = new Mocha({
     ui: "tdd",
     color: true,
+    rootHooks: {
+      beforeAll: async () => {
+        await sleep(5000); // Wait for the extension to be loaded
```

Comment:
What is your feeling around sleeps in the integration tests suite?

Is this just a necessary evil as we don't have other options? How brittle do you think we are on our integration tests?

Reply:
This `sleep` call only runs once before all the test suites, ensuring that they will have the features available. If there was a way to check that the LSP is fully initialized I'd use that but I didn't find a way to read extension state from the vscode API. Maybe I just didn't search enough but this does the trick.

Reply:
If the integration tests are reliable enough with this setup, lets hold off on finding other solutions.